includes/CMB2_hookup.php

```diff
@@ -339,15 +339,15 @@ public function add_context_metaboxes() {
 
 				// Call our before hook.
 				if ( ! empty( $this->cmb->prop( 'title' ) ) ) {
-					$this::context_box_title_markup_open( $this->cmb );
+					$this->context_box_title_markup_open( $this->cmb );
```

Comment:
if you're not going to use the `$cmb` variable passed in the function, can be removed from here